fares-data-build-tool/fdbt-site
# Feature/tfn 390 fare triangle ui improvements

# Description

This PR amends the styling attached to the fare triangle rendered on the priceEntry page. The background colour of the cells has been changed so that each row alternates in colour between white and light-grey. The contrast between the text and background is WCAG AA compliant as per GDS guidelines (contrast ratio is 18.78:1 versus minimum of 4.5:1). The width of the labels has also been adjusted to be slightly wider in order to accommodate longer stage names without showing an ellipsis.

# Testing instructions

-   Pull down this branch and run the site using the fdbt-dev repo.
-   Navigate to the priceEntry page and verify that (i) alternating cells are different colours and (ii) the labels accommodate a fare stage name of suitable length.

# Type of change


# Checklist:


[priceEntry.pdf](https://github.com/fares-data-build-tool/fdbt-site/files/4581893/priceEntry.pdf)
